src/test/java/seedu/address/model/JeeqTrackerTest.java

```diff
@@ -49,47 +49,47 @@ public void resetData_withDuplicateCompanies_throwsDuplicateCompanyException() {
         Company editedAlice = new CompanyBuilder(ALICE).withAddress(VALID_ADDRESS_BOB).withTags(VALID_TAG_HUSBAND)
                 .build();
         List<Company> newCompanies = Arrays.asList(ALICE, editedAlice);
-        AddressBookStub newData = new AddressBookStub(newCompanies);
+        JeeqTrackerStub newData = new JeeqTrackerStub(newCompanies);
 
-        assertThrows(DuplicateCompanyException.class, () -> addressBook.resetData(newData));
+        assertThrows(DuplicateCompanyException.class, () -> jeeqTracker.resetData(newData));
     }
 
     @Test
     public void hasCompany_nullCompany_throwsNullPointerException() {
-        assertThrows(NullPointerException.class, () -> addressBook.hasCompany(null));
+        assertThrows(NullPointerException.class, () -> jeeqTracker.hasCompany(null));
     }
 
     @Test
-    public void hasCompany_companyNotInAddressBook_returnsFalse() {
-        assertFalse(addressBook.hasCompany(ALICE));
+    public void hasCompany_companyNotInJeeqTracker_returnsFalse() {
+        assertFalse(jeeqTracker.hasCompany(ALICE));
     }
 
     @Test
-    public void hasCompany_companyInAddressBook_returnsTrue() {
-        addressBook.addCompany(ALICE);
-        assertTrue(addressBook.hasCompany(ALICE));
+    public void hasCompany_companyInJeeqTracker_returnsTrue() {
+        jeeqTracker.addCompany(ALICE);
+        assertTrue(jeeqTracker.hasCompany(ALICE));
     }
 
     @Test
-    public void hasCompany_companyWithSameIdentityFieldsInAddressBook_returnsTrue() {
-        addressBook.addCompany(ALICE);
+    public void hasCompany_companyWithSameIdentityFieldsInJeeqTracker_returnsTrue() {
+        jeeqTracker.addCompany(ALICE);
         Company editedAlice = new CompanyBuilder(ALICE).withAddress(VALID_ADDRESS_BOB).withTags(VALID_TAG_HUSBAND)
```

Comment:
Perhaps could consider changing the tags to suit the context of the company

Reply:
OK. I think we should do this in the future commits as there are a lot of test cases to change and it would be better to look through the test cases again when we are making changes to the classes at the same time.